### PR TITLE
Reformat DG User Stories Table 

### PR DESCRIPTION
**Pull request by Eugenetanwl3881**
Update all user stories to 

- Follow the format of "As a... I can... So that"
- Convert to table form

### CODE REVIEW

**Review by RichDom2185 — CHANGES_REQUESTED** [review]
This will unfortunately give a compile error on the website: all your individual table cell files have no `.md` extension, but the include statements have them.

Moreover, I don't think this is the correct approach. Note the following:
* Table headers are automatically bolded in markdown, no extra `**` necessary
* First column can be centre-aligned
* The table consists of simple statements, so using the following would be better

```markdown
| Priority |    As a             |  I can   |   So That     |
|:--------:|---------------------|----------|---------------|
|   `***`  | purchasing manager  | add..... | I can keep... |
```

File: docs/_dg/UserStories.md [hunk 1]
```diff
@@ -1,47 +1,31 @@
 <!-- markdownlint-disable-file first-line-h1 -->
 
-**Add Inventory Item:**
+Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
```
Comment (RichDom2185) [inline docs/_dg/UserStories.md:3]: There seems to be an inconsistency here, do we use `* * *` or `***` for high priority?

**Review by RichDom2185 — COMMENTED** [review]
Thanks for the reformatting! Just two questions: The first one below, and the second:

I noticed 23 lines in the markdown table, but originally there were 24 user stories. Were some combined together?

File: docs/_dg/UserStories.md (hunk 1 above)
Comment (Eugenetanwl3881) [reply docs/_dg/UserStories.md:3]: Ok will standardize to `* * *`

**Comment by Eugenetanwl3881** [comment]
> Thanks for the reformatting! Just two questions: The first one below, and the second:
> 
> I noticed 23 lines in the markdown table, but originally there were 24 user stories. Were some combined together?

Yes, there were repeates (search items by tags), thus I removed one.

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**